### ISSUE TITLE
Add frontend checks to CI pipeline

### ISSUE THREAD

**Issue by nwisnie** [nwisnie/Erg-lytics#98]
To close this issue, add automated ADA/accessibility (a11y) checks to our CI pipeline so every PR is tested for common WCAG/ADA violations in the frontend before it can be merged.

Axe-core + Playwright will probably be a good way to go about this.
